src/main/java/com/esri/samples/portal/oauth/README.md

```diff
@@ -1,42 +1,41 @@
-<h1>OAuth</h1>
+<h1>Authenticate with OAuth</h1>
 
-<p>Authenticate with OAuth 2.0 to retrieve the user's profile information.</p> 
+<p>Authenticate with ArcGIS Online (or your own portal) using OAuth2 to access secured resources (such as private web maps or layers). Accessing secured items requires logging in to the portal that hosts them (an ArcGIS Online account, for example).</p>
```

Comment:
I recognise this is still under design for README refactorings. This section should be max one sentence long and a short summary of what the app is primarily for. 

e.g. I want to.... "Use OAuth2 to authenticate with ArcGIS Online (or your own portal) to access secured resources "